packages/ios_platform_images/example/ios/RunnerTests/IosPlatformImagesTests.swift

```diff
@@ -0,0 +1,107 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+import XCTest
+
+@testable import ios_platform_images
+
+class IosPlatformImagesTests: XCTestCase {
+
+  var mockChannel: MockMethodChannel!
```

Comment:
Why do this and setUp exist? You're already initializing the mock in every test anyway (which is the preferred approach).

Reply:
I forgot remove him